### DISCUSSION TITLE
Change RGB image to CMYK image and upload it to Wordpress Library

### DISCUSSION THREAD

**Discussion by ggarciadms** [ImageMagick/ImageMagick#3721] [Help]
Hi, so as I said in the title I want to know if there is a way to change the color format of and image that is RGB to CMYK and then upload it to Wordpress Library and have an URL to use that image.

Any help could be handy, 

Thanks.

**Comment by fmw42** [comment]
ImageMagick can change the color format from RGB to CMYK. The uploading to Wordpress would have to be done by you.
